pkg/bigquery/utils/utils.go

```diff
@@ -0,0 +1,80 @@
+package utils
```

Comment:
**opinion (non-blocking)**: I'm not a big fan of `utils` packages, since it doesn't tell me much what to expect to find inside it. The [golang blog](https://go.dev/blog/package-names) has a good article on package names.

I would break this package down into two, maybe `httputils` and `dbutils`, or something like that

Reply:
This is a really good hint, thanks!

Reply:
@kirederik I'm merging this not to block @zoltanbedi and will fix in a followup pr.